### PR TITLE
carpool cancel emails

### PR DESCRIPTION
**Pull request by agdt3**
Mail host set through env vars.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -344,3 +363,44 @@ def cancel_carpool(carpool_id):
...
+    if send:
+        with mail.connect() as conn:
+            for rider in riders:
+                msg = Message(recipients=[rider.email],
+                              body=body,
+                              subject=subject)
+                conn.send(msg)
+    else:
+        for rider in riders:
+            print 'sent message to {} with subject {} and body {}'.format(
```
Comment (iandees) [inline app.py]: Can you use `current_app.logger.info(...)` instead of print?

File: app.py [hunk 2]
```diff
@@ -334,6 +347,10 @@ def cancel_carpool(carpool_id):
     cancel_form = CancelCarpoolDriverForm()
     if cancel_form.validate_on_submit():
         if cancel_form.submit.data:
+            _email_carpool_cancelled(
+                carpool_id,
```
Comment (iandees) [inline app.py]: Instead of passing in the `carpool_id`, can you pass in `carpool` so the emailer doesn't have to do another lookup?

File: app.py [hunk 3]
```diff
@@ -344,3 +361,44 @@ def cancel_carpool(carpool_id):
...
+    riders = carpool.riders
+    if len(riders) == 0:
+        return
+
+    if not reason:
+        reason = 'Reason not given!'
+
+    subject = 'Carpool session on {} cancelled'.format(carpool.leave_time)
+
+    body = '''
```
Comment (iandees) [inline app.py]: We might not want to do it in this pull request, but these emails should probably come from templates.

Comment (agdt3) [inline app.py]: Yep, I agree. I should have added a TODO.

File: app.py (hunk 2 above)
Comment (agdt3) [inline app.py]: Good point! Will change.

File: app.py (hunk 1 above)
Comment (agdt3) [inline app.py]: Will do!

**Pull request merged**